### DISCUSSION TITLE
no Events in SOC

### DISCUSSION THREAD

**Discussion by bienio20** [Security-Onion-Solutions/securityonion#11310] [2.4]
### Version

2.4.0

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Import

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

2

### RAM

8

### Storage for /

33

### Storage for /nsm

61

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello,
So after installing fresh SecurityOnion2.4 My SOC alerts are blank. I've been using simple test to trigger from other machine on the network but did not show any alerts. Any Idea where start to dig to resolve that? Thank you.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
Please follow the Troubleshooting Alerts section of the documentation:
https://docs.securityonion.net/en/2.4/suricata.html#troubleshooting-alerts

**Reply by tsmullins** [reply]
On your manager, check the logstash log in /opt/so/log/logstash/logstash.log for errors.

**Reply by bienio20** [reply]
There is no folder neither .log file named logstash in /opt/so/log. There is suricata folder but it is empty... Security Onion is installed on VM set to bridge and I added my local network. What next should I check?

**Reply by dougburks** [reply]
Per [my reply](https://github.com/Security-Onion-Solutions/securityonion/discussions/11310#discussioncomment-7005848) above, have you followed the Troubleshooting Alerts section of the documentation? I've copied the steps here for your convenience:

- If you have metadata enabled, check to see if you have metadata for the connections. Depending on your configuration, this could be Suricata metadata or [Zeek](https://docs.securityonion.net/en/2.4/zeek.html#zeek) metadata.
- If you have metadata enabled but aren’t seeing any metadata, then something may be preventing the process from seeing the traffic. Check to see if you have any [BPF](https://docs.securityonion.net/en/2.4/bpf.html#bpf) configuration that may cause the process to ignore the traffic. If you’re sniffing traffic from the network, verify that the traffic is reaching the NIC using tcpdump. If importing a pcap file, verify that file contains the traffic you expect and that the Suricata process can read the file and any parent directories.
- Check your HOME_NET configuration to make sure it includes the networks that you’re watching traffic for.
- Check to see if you have a full NIDS ruleset with rules that should specifically alert on the traffic and that those rules are enabled.
- Check to see if you have any threshold or suppression configuration that might be preventing alerts.
- Check the Suricata log for additional clues.
- Check the [Elastic Agent](https://docs.securityonion.net/en/2.4/elastic-agent.html#elastic-agent), [Logstash](https://docs.securityonion.net/en/2.4/logstash.html#logstash), and [Elasticsearch](https://docs.securityonion.net/en/2.4/elasticsearch.html#elasticsearch) logs for any pipeline issues that may be preventing the alerts from being written to [Elasticsearch](https://docs.securityonion.net/en/2.4/elasticsearch.html#elasticsearch).
- Try installing a simple import node (perhaps in a VM) following the steps in the [First Time Users](https://docs.securityonion.net/en/2.4/first-time-users.html#first-time-users) section and see if you get alerts there. If so, compare the working system to the non-working system and determine where the differences are.
